teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization, age, name: 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] Since you are using the colon (:) notation for your parameters, you need to use them correctly for your Parent class assignments or it will throw errors.

  ```
  super(age, name: name, parent_permission: parent_permission)
  ```